examples/logs-plugin/logs.conf

```diff
@@ -6,3 +6,7 @@ loggers:
    level: debug
  - name: "examplechildLogger"
    level: debug
+hooks:
+  syslog:
+    levels:
+    - panic
```

Comment:
Example should show use case with multiple levels.

Reply:
Ideally, panic/fatal/error, since first two of those will stop the program.

Reply:
An example is in cn-infra/logging/logging.conf

Reply:
That's not example of program.. it's an example for configuration. I want to run this example to see how the hooks work and what I can do with them.

Also, perhaps new example should be defined for hooks. Please copy this one and name it `logs-hooks-plugin`.